src/iop/clipping.c

```diff
@@ -1348,6 +1348,24 @@ void gui_focus(struct dt_iop_module_t *self, gboolean in)
 {
   dt_iop_clipping_gui_data_t *g = (dt_iop_clipping_gui_data_t *)self->gui_data;
   dt_iop_clipping_params_t *p = (dt_iop_clipping_params_t *)self->params;
+
+  // check if crop module is activated and show an alert if it's the case
+  // even if c&r is not still activated
+  if(in)
+  {
+    for(GList *l = g_list_last(self->dev->history); l; l = g_list_previous(l))
+    {
+      dt_dev_history_item_t *item = (dt_dev_history_item_t *)l->data;
+      if(!g_strcmp0(item->op_name, "crop"))
+      {
+        if(item->enabled)
+          dt_control_log(
+              _("cropping image in both 'crop' and 'crop &amp; rotate' modules may produce unexpected results"));
```

Comment:
In such case why not use the alert message circuitry already available? I'm talking about what we do for color calibration and white balance?

Reply:
you're right. PR updated to use in-module trouble message...